theme.qml

```diff
@@ -177,6 +177,14 @@ FocusScope {
     // Text strings
     property variant dataText: {
         "en": {
+            global_dimm: "dimm",
+            global_bright: "bright",
+            global_Dark: "Dark",
+            global_Light: "Light",
+            global_OzoneDark: "OzoneDark",
+            global_SteamOS: "SteamOS",
+            global_mono: "mono",
+            global_color: "color",

```

Comment:
```suggestion
            settings_collection_accentColor_dimm: "dimm",
            settings_collection_accentColor_bright: "bright",
            settings_general_colorScheme_Dark: "Dark",
            settings_general_colorScheme_Light: "Light",
            settings_general_colorScheme_OzoneDark: "OzoneDark",
            settings_general_colorScheme_SteamOS: "SteamOS",
            settings_general_logoVariant_mono: "mono",
            settings_general_logoVariant_color: "color",
```
global is for text strings which can appear in many places in the theme. I assume these strings will never be needed outside the settings screen. Therefore I would have adapted the naming scheme to this one

Reply:
ok good information